contractTest/gateways/MsSQL/retrieveDepartmentByCode.contractTest.js

```diff
@@ -0,0 +1,44 @@
+import retrieveDepartmentByCodeGateway from "../../../src/gateways/MsSQL/retrieveDepartmentByCode";
+import { setupOrganisationFacilityDepartmentAndManager } from "../../../test/testUtils/factories";
+import AppContainer from "../../../src/containers/AppContainer";
+
+describe("retrieveActiveDepartmentsByFacilityIdGateway", () => {
+  const container = AppContainer.getInstance();
+  it("retrieves department returning object containing trustId, wardId and wardCode", async () => {
+    // Arrange
+    const email = `${Math.random()}@nhs.co.uk`;
+    const departmentCreated = {
+      code: "WardCodeOne",
+    };
+    const {
+      orgId,
+      departmentId,
+      departmentUuid,
+    } = await setupOrganisationFacilityDepartmentAndManager({
+      userArgs: { email },
+      organisationArgs: {},
+      facilityArgs: {},
+      departmentArgs: { code: departmentCreated.code },
+    });
+    // Act
+    const department = await retrieveDepartmentByCodeGateway(container)(
+      departmentCreated.code
+    );
+    // Assert
+    expect(department).toEqual({
+      wardId: departmentId,
+      uuid: departmentUuid,
+      trustId: orgId,
+      wardCode: departmentCreated.code,
+      wardStatus: 1,
```

Comment:
This line is unclear, it'd be fine to make use of a lookup table for this value such as what I've done here: https://github.com/madetech/nhs-virtual-visit/pull/803/files#diff-75191d314c7ca43884e00799166d15af75d459019461dea0e93713837c0f138aR12-R27

Reply:
Good idea, please have a look at my pm to you, for further suggestions for this lookup table. Let me know what you think.

Reply:
This has been done